packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -5927,406 +5332,102 @@ export default class Meeting extends StatelessWebexPlugin {
       LoggerProxy.logger.log(
         `Meeting:index#processNextQueuedMediaUpdate --> performing delayed media update type=${mediaUpdateType}`
       );
+      let mediaUpdate = Promise.resolve();
+
       switch (mediaUpdateType) {
-        case MEDIA_UPDATE_TYPE.ALL:
-          this.updateMedia(options).then(pendingPromiseResolve, pendingPromiseReject);
-          break;
-        case MEDIA_UPDATE_TYPE.AUDIO:
-          this.updateAudio(options).then(pendingPromiseResolve, pendingPromiseReject);
-          break;
-        case MEDIA_UPDATE_TYPE.VIDEO:
-          this.updateVideo(options).then(pendingPromiseResolve, pendingPromiseReject);
-          break;
-        case MEDIA_UPDATE_TYPE.SHARE:
-          this.updateShare(options).then(pendingPromiseResolve, pendingPromiseReject);
+        case MEDIA_UPDATE_TYPE.TRANSCODED_MEDIA_CONNECTION:
+          mediaUpdate = this.updateTranscodedMediaConnection();
           break;
         case MEDIA_UPDATE_TYPE.LAMBDA:
```

Comment:
suggestion: LAMBDA is used twice, and both times it is used to call requestScreenShareFloor. This suggests that there isn't a need for a arbitrary type LAMBDA for the media update queue and instead could be a SHARE type.

Reply:
It can wait until the next PR